pkg/KubeArmorOperator/cmd/snitch-cmd/main.go

```diff
@@ -130,6 +135,14 @@ func snitch() {
 		Logger.Errorf("Not able to detect runtime")
 		os.Exit(1)
 	}
+	ociHooksLabel := "no"
+	if runtime == "cri-o" || EnableOCIHooks {
+		ociHooksLabel = "yes"
+		if err := applyCRIOHook(socket); err != nil {
```

Comment:
just to confirm, it's genric way to check OCI hook support for any CRI not limited to cri-o right!

Reply:
so do we need to handle anything here?